index.html

```diff
@@ -46,7 +46,7 @@
         <div class="nav-container">
```

Comment:
For the fonts above, I don't think we need the crossorigin attributes since these are local files (I noticed some console warnings regarding these).

Reply:
Thank you! I already deleted the crossorigin attributes and it deleted those warnings on the console.